subjects/devops/grades/README.md

```diff
@@ -0,0 +1,108 @@
+## grades
+
+### Instructions
+
+Create a script `grades.sh` which will ask the user for a specific number of students in order to evaluate them. After the number is set, the script will ask you to introduce the "names" and "grades" of the persons you wish to evaluate.
+
+The grades will have a range between 0 and 100 and will be numeric values only.
+
+According to each student grade you will have to return the following:
+
+- If the student grade is anything lower than 50 you will return the string "<name>: You had a poor performance!":
+
+```console
+$ ./grades.sh 1
+Student Name #1: Sara
+Student Grade #1: 34
+Sara: You had a poor performance!
+```
+
+- If the student grade is anything equal or greater than 50 you will return the string "<name>: You need a bit more effort!":
+
+```console
+$ ./grades.sh 1
+Student Name #1: Sara
+Student Grade #1: 51
+Sara: You need a bit more effort!
+```
+
+- If the student grade is anything equal or greater than 70 you will return the string "<name>: You did a good job!":
+
+```console
+$ ./grades.sh 1
+Student Name #1: Sara
+Student Grade #1: 75
+Sara: You did a good job!
+```
+
+- If the student grade is anything equal or greater than 90 you will return the string "<name>: You did an excellent job!":
+
+```console
+$ ./grades.sh 1
+Student Name #1: Sara
+Student Grade #1: 90
+Sara: You did an excellent job!
+```
+
+### Error handling
+
+All errors will print a specific message on **stderr** (ending with a newline) and returns a specific non-zero value:
+
+- Wrong number of arguments: `"Error: expect 2 arguments"`, exit with `1`.
+
+- If the student grade is not a number or is greater than 100: `Error: The grade "grade" is not a valid input. Only numerical grades between 0 and 100 are accepted.`, exit with `1`.
+
+```console
```

Comment:
maybe we can add also an example with wrong number of arguments here

Reply:
done